### ISSUE TITLE
Ability to run validation on last epoch, even when using `check_val_every_n_epoch`

### ISSUE THREAD

**Issue by catalys1** [PyTorchLightning/pytorch-lightning#4539]
## 🚀 Feature
The ability to run a validation loop after the very last training epoch, even when `check_val_every_n_epoch > 1` and the last epoch doesn't coincide with `current_epoch % check_val_every_n_epoch == 0`.

### Motivation

Sometimes it's nice to run validation less often, for instance to speed up training, but I usually want to see the final validation accuracy even if I've skipped it on some of the intermediate epochs.

### Pitch

I'm not sure what the best way to implement this feature is; perhaps another flag in `Trainer`, although there are already quite a few there.

**Comment by tchaton** [comment]
Hey @catalys1,

Thanks for sharing the idea ! I like it !
I think we should make it a default in the code, so it doesn't add a non-essential extra arguments.

Would you like to give it a try ?

Best regards,
Thomas Chaton.

**Comment by catalys1** [comment]
Thanks @tchaton, sounds good. So just to make sure I'm understanding correctly, the change would be to ALWAYS run validation loop after last training loop (not making it a flag/optional)? If so, that seems fine to me.

I can take a look, but I might not be able to get to it until after next week.

**Comment by edenlightning** [comment]
@tchaton ccomment?

**Comment by sanketsans** [comment]
@catalys1 Great thought. I can try for this one. Will be a nice first issue for me.

**Comment by catalys1** [comment]
@sanketsans If you want to tackle it, feel free!

**Comment by manbldwin** [comment]
New to me

**Comment by yitianwu-motional** [comment]
Hi there, is there any solution for this issue? @edenlightning

**Comment by timothylimyl** [comment]
Why can the issue be closed without reference to any PR?
